src/pages/jaehyeon/Main/Comment.js

```diff
@@ -0,0 +1,51 @@
+// import { useState } from 'react';
+// import './Comment.scss';
+import profile from '../../../assets/jaehyeon/profile.jpg';
+
+function Comment({
+  commentValue,
+  comment,
+  setComment,
+  i,
+  isLike,
+  setIsLike,
+  heart,
+  setHeart,
+  visitorName,
+  visitorImg,
+}) {
+  return (
+    <li className="comment">
+      <div className="comment-item">
+        <div className="comment-profile">
+          <img src={visitorImg ? visitorImg : profile} alt="" />
+        </div>
+        <span className="idText">{visitorName ? visitorName : 'Jaehyeon'}</span>
+        <div className="comment-output-text">{commentValue}</div>
+      </div>
+      <div className="commentHeart">
+        <i
+          className={isLike[i] ? 'fa-solid fa-heart' : 'fa-regular fa-heart'}
+          onClick={() => {
+            let copy = [...isLike];
+            copy[i] = !copy[i];
+            setIsLike(copy);
+          }}
+        />
+        <button
+          className="remove-button"
+          disabled={!visitorImg ? false : true}
```

Comment:
```suggestion
          disabled={!visitorImg}
```
visitorImg라는 값 자체가 boolean 값을 갖고 있으니 이렇게만 넣어줘도 같은 방식으로 동작합니다!